packages/run/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@waku/run",
+  "version": "0.0.1",
+  "description": "Run a local Waku network for development and testing",
+  "type": "module",
+  "author": "Waku Team",
+  "homepage": "https://github.com/waku-org/js-waku/tree/master/packages/run#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/waku-org/js-waku.git"
+  },
+  "bugs": {
+    "url": "https://github.com/waku-org/js-waku/issues"
+  },
+  "license": "MIT OR Apache-2.0",
+  "keywords": [
+    "waku",
+    "decentralized",
+    "communication",
+    "web3",
+    "testing",
+    "development"
+  ],
+  "bin": {
+    "waku-run": "./dist/src/cli.js"
+  },
+  "files": [
+    "dist",
+    "docker-compose.yml",
+    "init-db.sh",
+    "README.md"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "build:web": "esbuild web/index.ts --bundle --format=esm --platform=browser --outdir=dist/web && cp web/index.html dist/web/index.html",
+    "prepublishOnly": "npm run build",
+    "start": "tsx scripts/start.ts",
+    "stop": "tsx scripts/stop.ts",
+    "restart": "npm run stop && npm run start",
+    "logs": "tsx scripts/logs.ts",
+    "info": "tsx scripts/info.ts",
+    "test": "if [ \"$CI\" = \"true\" ]; then echo 'Skipping tests in CI'; exit 0; fi && NODE_ENV=test node ./src/run-tests.js \"tests/basic.spec.ts\"",
+    "test:browser": "if [ \"$CI\" = \"true\" ]; then echo 'Skipping browser tests in CI'; exit 0; fi && npm run build:web && npx playwright test --reporter=line",
+    "fix": "run-s fix:*",
+    "fix:lint": "eslint src scripts tests web --fix",
+    "check": "run-s check:*",
+    "check:tsc": "tsc -p tsconfig.dev.json",
+    "check:lint": "eslint src scripts tests web",
+    "check:spelling": "cspell \"{README.md,src/**/*.ts,scripts/**/*.ts,tests/**/*.ts}\""
+  },
+  "engines": {
+    "node": ">=18"
+  },
+  "dependencies": {
+    "@waku/core": "*",
+    "@waku/interfaces": "*",
+    "@waku/sdk": "*",
+    "@waku/utils": "*"
+  },
+  "devDependencies": {
+    "@playwright/test": "^1.51.1",
+    "@types/chai": "^4.3.11",
+    "@types/express": "^4.17.21",
+    "@types/mocha": "^10.0.6",
+    "chai": "^4.3.10",
+    "cspell": "^8.6.1",
+    "esbuild": "^0.21.5",
+    "express": "^4.21.2",
+    "mocha": "^10.3.0",
+    "npm-run-all": "^4.1.5",
+    "ts-node": "^10.9.2",
+    "tsx": "^4.7.0",
```

Comment:
why do you need tsx?

Reply:
it was used to run scripts in package.json, but can be safely removed and use dist/... instead